src/System.Linq.Expressions/tests/ReadOnlyCollectionBuilder/ReadOnlyCollectionBuilder.Tests.Misc.cs

```diff
@@ -0,0 +1,560 @@
+                {
+                    list.Clear();
+                    Assert.Equal(list.Count, 0); //"Should be equal to 0."
+                }
+            }
+
+            public void NonGenericIListClearEmptyList()
+            {
+                ReadOnlyCollectionBuilder<T> list = new ReadOnlyCollectionBuilder<T>();
+                IList _ilist = list;
+                Assert.Equal(list.Count, 0); //"Should be equal to 0."
+                _ilist.Clear();
+                Assert.Equal(list.Count, 0); //"Should be equal to 0."
+            }
+            public void NonGenericIListClearMultipleTimesEmptyList(int times)
+            {
+                ReadOnlyCollectionBuilder<T> list = new ReadOnlyCollectionBuilder<T>();
+                IList _ilist = list;
+                Assert.Equal(list.Count, 0); //"Should be equal to 0."
+                for (int i = 0; i < times; i++)
+                {
+                    _ilist.Clear();
+                    Assert.Equal(list.Count, 0); //"Should be equal to 0."
+                }
+            }
+            public void NonGenericIListClearNonEmptyList(T[] items)
+            {
+                ReadOnlyCollectionBuilder<T> list = new ReadOnlyCollectionBuilder<T>(items);
+                IList _ilist = list;
+                _ilist.Clear();
+                Assert.Equal(list.Count, 0); //"Should be equal to 0."
+            }
+
+            public void NonGenericIListClearMultipleTimesNonEmptyList(T[] items, int times)
+            {
+                ReadOnlyCollectionBuilder<T> list = new ReadOnlyCollectionBuilder<T>(items);
+                IList _ilist = list;
+                for (int i = 0; i < times; i++)
+                {
+                    _ilist.Clear();
+                    Assert.Equal(list.Count, 0); //"Should be equal to 0."
+                }
+            }
+
+            #endregion
+
+            #region ToArray
+
+            public void BasicToArray(T[] items)
+            {
+                ReadOnlyCollectionBuilder<T> list = new ReadOnlyCollectionBuilder<T>(items);
+
+                T[] arr = list.ToArray();
+
+                for (int i = 0; i < items.Length; i++)
+                {
+                    Assert.Equal(((object)arr[i]), items[i]); //"Should be equal."
+                }
+            }
+
+            public void EnsureNotUnderlyingToArray(T[] items, T item)
+            {
+                ReadOnlyCollectionBuilder<T> list = new ReadOnlyCollectionBuilder<T>(items);
+                T[] arr = list.ToArray();
+                list[0] = item;
+                if (((object)arr[0]) == null)
+                    Assert.NotNull(list[0]); //"Should NOT be null"
+                else
+                    Assert.NotEqual(((object)arr[0]), list[0]); //"Should NOT be equal."
+            }
+
+            #endregion
+        }
+
+        [Fact]
+        public static void InsertTests()
+        {
+            Driver<int> IntDriver = new Driver<int>();
```

Comment:
Use `intDriver` instead; local variables should be camel-case. You can refer to the coding conventions [here](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md).